lib/crud_responder/locales/en.yml

```diff
@@ -0,0 +1,12 @@
+en:
```

Comment:
It's will copy to config/locales in rails app. But can you clarify me what do you mean?